packages/parcel-transformer/src/types.ts

```diff
@@ -47,4 +47,13 @@ export interface ParcelTransformerOpts extends BabelPluginOpts {
    * Default to `undefined`
    */
   classNameCompressionMap?: { [index: string]: string };
+
+  /**
+   * File path to classNameCompressionMap. It's relative to project root.
+   * Note `classNameCompressionMap` and `classNameCompressionMapFilePath` are mutually exclusive.
+   * If both are provided, classNameCompressionMap takes precedence.
+   *
+   * Default to `undefined`
+   */
+  classNameCompressionMapFilePath?: 'string';
```

Comment:
Not sure if `'string'` works here (I expected `string` without quotes)

Reply:
Ah yeah I think the quotes should be removed from `'string'`, otherwise `'string'` would be the only valid string value for `classNameCompressionMapFilePath`

Reply:
> Not sure if `'string'` (with quotes) works here (I expected `string` without quotes)

Good catch!